utils/submit_tracks.py

```diff
@@ -24,29 +32,48 @@ class TrackData(TypedDict):
     trigger: list[str]
     type: str
 
-
-class CSVRow(TypedDict):
-    desc: str
-    name: str
-    sources: list[str]
-    urls: list[str]
-
-
-CSVData = dict[str, CSVRow]
-CSVCollection = dict[str, CSVData]
+# Datamodel for track descriptions
+class DescData(TypedDict):
```

Comment:
Track descriptions seem to be just one field of this object. Same for the rest! This seem to be DBTrackData

Reply:
It's mostly description data (source/url is part of track description) and hte data is sourced both from DB (gene tracks) and CSV (variation tracks), so DBTrackData is not a good name either.

Reply:
If that's the case why we're not using TrackDesc object anywhere else?